guides/plugins/apps/custom-data/custom-entities.md

```diff
@@ -0,0 +1,52 @@
+# Custom entities
+
+You can add custom entities to Shopware and thus extend the functionality of your shop.
```

Comment:
can you add a little bit more explanatory text on where to use the feature?

E.g. if you want to store custom complex data in the shopware DB you can use custom entities to do so...